README.md

```diff
@@ -581,6 +583,30 @@ This document is written to capture the conventions of writing styles for our pr
 ### Disclaimer
 It's important to note that client and project requirements always undo choices made in these guidelines
 
+
+### Styleguide implementation
+
+#### Colors
+Store colors centrally as variables. Preferably use human-friendly names that make communication between departments easier. In many cases clients will provide brand guidelines with names for colors. Use these when possible in all documentation and communication.
```

Comment:
@nswaldman you can use the insert suggestion thingy to make it easier for the author to fix feedback 😉  It will also add you as a co-author to the commit iirc, this can be helpful when when you look at the git history.

![image](https://user-images.githubusercontent.com/5376407/139655361-79079fb2-8f2e-4c7a-85a7-96fcc40a1cef.png)

Example:

```suggestion
Store colours as variables and prefer human-readable names to ease communication between disciplines. Clients may provide additional naming conventions based on brand guidelines. Prefer these in all communication.
```

Reply:
I think that we should a bit more specific in our explanations, so that the person reading the standards already has an idea of what they have to do.

```suggestion
We _must not_ maintain a separate set of naming conventions as it might make communication between departments and client confusing. Color naming should be aligned with either the design team, brand guidelines or client provided conventions.

All colors should be stored in a globally accessible manner and declared following the guidelines of CSS or the chosen CSS framework: 

- for CSS - [custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties),
- for SASS/SCSS - [variables](https://sass-lang.com/documentation/variables),
- for CSS-in-JS - depends on the feature-set:
  - if theming is supported, the colors should be stored inside of a theme,
  - otherwise a TypeScript `enum`.
```